pkg/test/verify/cluster.go

```diff
@@ -181,9 +190,7 @@ func UpdateToolchainCluster(t *testing.T, functionToVerify FunctionToVerify) {
 	require.True(t, ok)
 	assert.Equal(t, statusFalse, *cachedToolchainCluster.ClusterStatus)
 	AssertClusterConfigThat(t, cachedToolchainCluster.Config).
-		IsOfType(cluster.Host).
 		HasName("east").
-		HasOperatorNamespace("toolchain-host-operator").
```

Comment:
I think we still want to check the namespace ?

Reply:
yes, missed it somehow, adding it back again